app/models/manageiq/providers/azure/inventory/parser/cloud_manager.rb

```diff
@@ -83,6 +84,7 @@ def instances
         :raw_power_state     => status,
         :flavor              => series,
         :location            => instance.location,
+        :parent              => persister.miq_templates.lazy_find(parent_ref),
```

Comment:
it has to be called ```genealogy_parent```, there is after save logic handling this as backup, but we use custom saver saving this